client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManager.java

```diff
@@ -55,7 +55,7 @@
  * @author Mircea.Markus@jboss.com
  * @since 4.1
  */
-public class RemoteCacheManager implements BasicCacheContainer {
```

Comment:
There is a small dichotomy here... 

`EmbeddedCacheManager` is an interface and for remote one we use `RemoteCacheContainer`. I think it would make sense to rename `RemoteCacheContainer` -> `RemoteCacheManager` and `RemoteCacheManager` -> `Default?RemoteCacheManager`. If you won't do it now - we'll stick to this dichotomy for whole ISPN 9 time frame.